13/alpine3.10/Dockerfile

```diff
@@ -12,7 +12,7 @@ RUN addgroup -g 1000 node \
       && case "${alpineArch##*-}" in \
         x86_64) \
           ARCH='x64' \
-          CHECKSUM="b5b1af4f97bf8759e0de58e0d659ffa81db4857500e56e2b0b3f5948351a9bfa" \
+          CHECKSUM="" \
```

Comment:
The unofficial builds aren't ready yet, just triggering this to see if the Chakracore cleanup messed the build up